### ISSUE TITLE
./fab installation error: [Jaeger] "updating chart dependencies failed with:   "

### ISSUE THREAD

**Issue by arnol229** [CatalystCode/kubemalt#33]
Running ./fab install produces error when going through jaeger installation:
```
ERRO[05-02-2019 12:20:56] updating chart dependencies failed with:   
```

**Comment by mtarng** [comment]
Removing Jaeger from the component.yaml fixes this build on fab version 0.1.4. Haven't investigated why.

**Comment by mtarng** [comment]
Added missing step to the wiki: https://github.com/CatalystCode/kubemalt/wiki/Various-helpful-docs-and-bash-scripts-for-Kubernetes-and-Docker-deployments#fabrikate-deployment-commands

We need to add a missing helm repo first before running fab install.
